subsys/infinispan/src/main/resources/infinispan-jdbc.xml

```diff
@@ -7,13 +7,14 @@
 
   <cache-container default-cache="local" name="IndyCacheManager" shutdown-hook="DEFAULT" statistics="true">
     <local-cache-configuration name="local-template" statistics="true">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
+      <memory>
+        <object size="200000" strategy="LRU" />
+      </memory>
     </local-cache-configuration>
 
     <local-cache name="local" configuration="local-template"/>
 
     <local-cache name="koji-maven-version-metadata" configuration="local-template">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
```

Comment:
For stored metadata, we actually do need to set the memory section, and set the number *much* lower than 20k objects. I checked our production database the other day, and the caches holding maven metadata were each consuming 30GB on disk, without us hitting this maximum number AFAIK.

Reply:
Add mem control to 2000